### PR TITLE
Added python version selection page

### PR DESCRIPTION
**Pull request by TDaglis**
Added instructions for changing the default python and pip version

### CODE REVIEW

File: _runtime/python-version.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: Choose python version in SourceLair
```
Comment (parisk) [inline _runtime/python-version.md]: I would change "Choose python version in SourceLair" to "Choosing Python version". Looks cleaner to me and I think we have overwhelmed our help page with the word "SourceLair".

In, any case please change "python" to "Python".

File: _runtime/python-version.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+layout: page
+---
+
+In SourceLair, the **default** python version is **2.7.6**. However, you are able to choose between python 2.7.6 and 3.4.3.
```
Comment (parisk) [inline _runtime/python-version.md]: Please change "python" to "Python".

File: _runtime/python-version.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+In order to activate python **3.4.3** all you have to do is [paste](http://help.sourcelair.com/terminal/copy-paste/) the following command in your terminal:
```
Comment (parisk) [inline _runtime/python-version.md]: Please change "python" to "Python"

File: _runtime/python-version.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+title: Choose python version in SourceLair
+layout: page
+---
+
+In SourceLair, the **default** python version is **2.7.6**. However, you are able to choose between python 2.7.6 and 3.4.3.
+
+In order to activate python **3.4.3** all you have to do is [paste](http://help.sourcelair.com/terminal/copy-paste/) the following command in your terminal:  
+`mkdir -p /mnt/data/.local/bin && ln -fs $(which python3) /mnt/data/.local/bin/python && ln -fs $(which pip3) /mnt/data/.local/bin/pip`
+
+In order to activate python **2.7.6**, [paste](http://help.sourcelair.com/terminal/copy-paste/) the following command in your terminal:
```
Comment (parisk) [inline _runtime/python-version.md]: Please change "python" to "Python"

File: _runtime/python-version.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+
+In SourceLair, the **default** python version is **2.7.6**. However, you are able to choose between python 2.7.6 and 3.4.3.
+
+In order to activate python **3.4.3** all you have to do is [paste](http://help.sourcelair.com/terminal/copy-paste/) the following command in your terminal:  
+`mkdir -p /mnt/data/.local/bin && ln -fs $(which python3) /mnt/data/.local/bin/python && ln -fs $(which pip3) /mnt/data/.local/bin/pip`
+
+In order to activate python **2.7.6**, [paste](http://help.sourcelair.com/terminal/copy-paste/) the following command in your terminal:  
+`mkdir -p /mnt/data/.local/bin && ln -fs $(which python2) /mnt/data/.local/bin/python && ln -fs $(which pip2) /mnt/data/.local/bin/pip`
+
+Right after changing your python version make sure you **restart your server as well as your terminal** for the change to take effect. Any process (server, terminal, run file, etc) that starts after the change will run with the newly selected python version.
```
Comment (parisk) [inline _runtime/python-version.md]: Please change "python" to "Python"

File: _runtime/python-version.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+
+At any time, if you want to check the currently active python version you can run the following command in your terminal:
```
Comment (parisk) [inline _runtime/python-version.md]: Please change "python" to "Python"

File: _runtime/python-version.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+In order to activate python **3.4.3** all you have to do is [paste](http://help.sourcelair.com/terminal/copy-paste/) the following command in your terminal:  
+`mkdir -p /mnt/data/.local/bin && ln -fs $(which python3) /mnt/data/.local/bin/python && ln -fs $(which pip3) /mnt/data/.local/bin/pip`
+
+In order to activate python **2.7.6**, [paste](http://help.sourcelair.com/terminal/copy-paste/) the following command in your terminal:  
+`mkdir -p /mnt/data/.local/bin && ln -fs $(which python2) /mnt/data/.local/bin/python && ln -fs $(which pip2) /mnt/data/.local/bin/pip`
+
+Right after changing your python version make sure you **restart your server as well as your terminal** for the change to take effect. Any process (server, terminal, run file, etc) that starts after the change will run with the newly selected python version.
+
+At any time, if you want to check the currently active python version you can run the following command in your terminal:  
+`python -c 'import sys; print(sys.version_info[:])'`
```
Comment (parisk) [inline _runtime/python-version.md]: What about proposing our users to run `python --version` instead of this?

**Comment by parisk** [comment]
:+1: overall. Proposed a couple of content updates + uppercasing the first letter in "python".

Good job.

File: _runtime/python-version.md (hunk 7 above)
Comment (TDaglis) [inline _runtime/python-version.md]: sure :+1: 
I'm changing it

**Comment by TDaglis** [comment]
@parisk resolved comments

**Comment by parisk** [comment]
Awesome!

LGTM :cake:

**Pull request merged**